### PR TITLE
[Glossary] Add Twig filter

### LINKED ISSUE #7535: [Glossary] Twig apply filter
```twig 
{% apply glossary%}
        <div class="page-header">
            <h1>{{ pimcore_input('headline', {'placeholder':'Headline'}) }}</h1>
        </div>

        {{ pimcore_areablock('content') }}
{% endapply %}
```

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves #7535

### CODE REVIEW

**Comment by brusch** [comment]
I think we should also deprecate the `pimcoreglossary` tag here, wdyt?

**Comment by dvesh3** [comment]
> I think we should also deprecate the `pimcoreglossary` tag here, wdyt?

done.

File: doc/Development_Documentation/02_MVC/02_Template/02_Template_Extensions/README.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ All Twig extension functions are described below in detail, the following tables
 |------------------------------------------|-------------------------------------------------------------------|
 | `pimcore_cache()`         | Simple in-template caching functionality                                         |
 | `pimcore_device()`        | Helps implementing adaptive designs                                              |
-| `pimcoreglossary()`       | Extension to control the glossary engine                                         |
+| `pimcoreglossary()`       | Deprecated: Extension to control the glossary engine                             |
+| `pimcore_glossary`        | Twig Fitler: Apply filter on content to pass it to Glossary engine               |
```
Comment (cadeyrn) [inline doc/Development_Documentation/02_MVC/02_Template/02_Template_Extensions/README.md]: typo ("Fitler") 😉

Comment (dvesh3) [reply doc/Development_Documentation/02_MVC/02_Template/02_Template_Extensions/README.md]:
```suggestion
| `pimcore_glossary`        | Twig Filter: Apply filter on content to pass it to Glossary engine               |
```

Comment (dvesh3) [reply doc/Development_Documentation/02_MVC/02_Template/02_Template_Extensions/README.md]: Fixed. thanks!

**Pull request merged**